### PR TITLE
Set classifiers in setup.py.

### PR DESCRIPTION
**Pull request by junaruga**
It is as the title.
I did set seeing https://pypi.python.org/pypi?:action=list_classifiers

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,6 +30,13 @@
         'pytest-runner',
     ],
     classifiers=[
-        # TODO(Add items)
+        (
+            'License :: OSI Approved :: ',
+            'GNU General Public License v2 or later (GPLv2+)'
+        ),
```
Comment (hroncok) [inline setup.py]: I know we are trying to please flake8, but this is uglier than a long line. The parenthesis are not necessary within a list definition.

Comment (hroncok) [inline setup.py]: Also, with a comma, this is a tuple with two strs. I guess that was not intentional.

Comment (junaruga) [inline setup.py]: Thanks for mentioning it. I tried to do this http://python-guide-pt-br.readthedocs.io/en/latest/writing/style/#line-continuations . But I mistook it.
Rebased!

File: setup.py [hunk 2]
```diff
@@ -30,6 +30,11 @@
         'pytest-runner',
     ],
     classifiers=[
-        # TODO(Add items)
+        'License :: OSI Approved :: '
+        'GNU General Public License v2 or later (GPLv2+)'
```
Comment (hroncok) [inline setup.py]: now there is a comma missing

Comment (junaruga) [inline setup.py]: I am sorry. rebased with adding the comma.

**Comment by junaruga** [comment]
> You might want to add Programming Language :: Python :: 3 :: Only, but it's not necessary.

Thanks for the reviewing.
Yes, maybe. Additional PR from you, is welcome!

**Pull request merged**